packages/sync/src/modules/Comments.php

```diff
@@ -134,6 +134,27 @@ public function init_full_sync_listeners( $callable ) {
 		add_action( 'jetpack_full_sync_comments', $callable ); // Also send comments meta.
 	}
 
+	/**
+	 * Gets a filtered list of comment types that sync can hook into.
+	 *
+	 * @access public
+	 *
+	 * @return array Defaults to [ '', 'trackback', 'pingback' ].
+	 */
+	public function get_hookable_comment_types() {
```

Comment:
Just like any of our whitelist methods assume that they're hookable/filterable by default, I think we should be safe to assume that this one should be hookable by default too. 

Meaning, we could remove the `hookable` from the names everywhere. Especially from the filter name, since it being a filter implies that it's hookable. I think I'd use "whitelisted" instead.

What do you think?
```suggestion
	public function get_whitelisted_comment_types() {
```